src/Common/tests/System/Net/Http/LoopbackProxyServer.cs

```diff
@@ -43,6 +44,8 @@ public sealed class LoopbackProxyServer : IDisposable
         public List<ReceivedRequest> Requests => _requests;
         public Uri Uri => _uri;
 
+        public event ErrorEventHandler Errors;
```

Comment:
We typically don't expose public fields.  Instead we expose a public property that wraps a private field.

See the other properties "Connections" and "Requests" as examples.

That said, we haven't used `event` as a pattern in our test code before. At most, we would call FailFast() or use NetEventSource logging (use test event source).